kong/cmd/utils/migrations.lua

```diff
@@ -151,16 +160,18 @@ local function up(schema_state, db, opts)
 end
 
 
-local function finish(schema_state, db, ttl)
+local function finish(schema_state, db, opts)
   if schema_state.needs_bootstrap then
     log("can't run migrations: database not bootstrapped")
     return
   end
 
-  local opts = {
-    ttl = ttl,
-    no_wait = true, -- exit the mutex if another node acquired it
-  }
+  if opts.force then
+    log("cannot use --force with 'finish'")
```

Comment:
let's keep the language consistent: cannot vs can't (above). Which should pick one and stick with it (any of the two is fine imho).

Reply:
That module already had 2 `can't`s and 2 `cannot`s in other messages :) Will standardize them on `cannot` for :robot: computer voice :robot:  reasons, then

Reply:
Awesome!